mono/metadata/profiler.c

```diff
@@ -283,7 +283,7 @@ mono_profiler_install_monitor  (MonoProfileMonitorFunc callback)
 }
 
 static MonoProfileSamplingMode sampling_mode = MONO_PROFILER_STAT_MODE_PROCESS;
-static int64_t sampling_frequency = 1000; //1ms
+static int64_t sampling_frequency = 10000; //10ms
```

Comment:
Sure. I intend to document most of the profiler changes in the release notes as there quite a lot, and many of them do affect users.